client/src/ui/modules/rewards/Rewards.tsx

```
@@ -0,0 +1,190 @@
+        AddressName,
+        Leaderboard,
+        LeaderboardRegistered,
+        events: { GameEnded },
+      },
+      systemCalls: { register_to_leaderboard, claim_leaderboard_rewards },
+    },
+  } = useDojo();
+
+  const [timeRemaining, setTimeRemaining] = useState<string>("");
+
+  const [prizePool, setPrizePool] = useState<BigInt>(0n);
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)